integration/integration_test.go

```diff
@@ -276,6 +276,11 @@ func TestAuditSeccomp(t *testing.T) {
 
 	t.Parallel()
 
+	seccompArch, err := k8sArchToSeccompArch(*k8sArch)
```

Comment:
There are some arch conversions in [libseccomp.GetArchFromString()](https://github.com/seccomp/libseccomp-golang/blob/main/seccomp.go#L264-L266).

But it does not seem to be the ones you want. I wonder if the missing conversion function could be added in libseccomp-golang.

Or more easily, maybe just remove the `architectures` list completely, since it is optional?